lib/msal-browser/src/interaction_client/PlatformAuthInteractionClient.ts

```diff
@@ -956,7 +956,7 @@ export class PlatformAuthInteractionClient extends BaseInteractionClient {
             windowTitleSubstring: document.title,
             extraParameters: {
                 ...request.extraQueryParameters,
-                ...request.tokenQueryParameters,
+                ...request.extraParams,
```

Comment:
This can probably just be extraParams since WAM will not respect the query contract